HTML Boilerplate lesson: reword sentences (#24097)
TheOdinProject/curriculum: foundations/html_css/html-foundations/html-boilerplate.md

### VSCode Shortcut
VSCode has a built-in shortcut you can use for generating all the boilerplate in one go. Please note that this shortcut only works while editing a file with the '.html' extension or a text file with the HTML language already selected. To trigger the shortcut, delete everything in the `index.html` file and just enter `!` on the first line. This will bring up a couple of options. Press the enter key to choose the first one, and voila, you should have all the boilerplate populated for you.

> But it's still good to know how to write the boilerplate yourself in case you find yourself using a text editor like notepad (heaven forbid) which doesn't have this shortcut. Try not to use the shortcut in your first few HTML projects, so you can build some muscle memory after manually writing the boilerplate code.

Reviewer: I think your suggestion is saying something slightly different than originally intended here but I agree that the existing wording is a little bit clunky. How would you feel about changing it to the below suggestion?
```suggestion
But it's still good to know how to write the boilerplate yourself in case you find yourself using a text editor like notepad (heaven forbid) which doesn't have this shortcut. Try not to use the shortcut in your first few HTML projects, so you can build some muscle memory for writing the boilerplate code.
```
